tests/queries/0_stateless/02346_text_index_map_support.sql

```diff
@@ -0,0 +1,379 @@
+-- Tags: no-parallel-replicas
```

Comment:
Same comments as in the corresponding test for arrays apply to this file as well.
It would be good to simplify these tests and remove some repetiveness.